src/phases/execution/MultiDeletes.yml

```diff
@@ -0,0 +1,161 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Runs inserts on one collection while a mass deletion takes place on another collection in the same database.
+
+GlobalDefaults:
+  # Configurable parameters
+  DocumentSizePaddingBytes: &DocumentSizePaddingBytes {^Parameter: {Name: "DocumentSizePaddingBytes", Default: 0}}
+  DocumentCount: &DocumentCount {^Parameter: {Name: "DocumentCount", Default: 100_000_000}}
+
+# Static variables.
+WriteRate: &WriteRate 1 per 500 microseconds  # 2000/second
+Database: &Database test
+# Collection0 is the default collection for the MonotonicSingleLoader.
+MassDeletionsColl: &MassDeletionsColl Collection0
+
+Actors:
+
+# Phase 1: Drop the database.
+- Name: DropDatabase
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand: {dropDatabase: 1}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Populate the collection for mass deletion. TODO Make multithreaded once TIG-2938 is resolved.
+- Name: PopulateInitialData0
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: *Database
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: *DocumentCount
+    BatchSize: 1000
+    Document:
+      padding: {^FastRandomString: {length: *DocumentSizePaddingBytes}}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Gaurd against timout due to no logging during PrePoulation0.
+- Name: LoggingActor0
+  Type: LoggingActor
+  Threads: 1
+  Phases:
+  - *Nop
+  - LogEvery: 9 minutes
+    Blocking: None
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 3: Issue a large delete that should take a considerable amount of time to run. This delete
+# will run with no concurrent writes to the database.
+- Name: LargeMultiDelete0
+  Type: CrudActor
+  Database: *Database
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    MetricsName: DuringMassDeletion0
+    Collection: *MassDeletionsColl
+    Operations:
+    - OperationName: bulkWrite
+      OperationCommand:
+        WriteOperations:
+        - WriteCommand: deleteMany
+          Filter: {_id: {$gte: 0}}
+          OperationOptions:
+            WriteConcern: { Level: "majority" }
+  - *Nop
+  - *Nop
+
+# Phase 4: Populate the collection for mass deletion again. This time concurrent writes will be
+# issued to another collection in the same database. TODO Make multithreaded once TIG-2938 is resolved.
+- Name: PopulateInitialData1
+  Type: Loader
```

Comment:
Update: we believe this is because normal index scans have type bounds. Which means GTE _id: 0 won't incorporate id's with type OID (see [key_string.cpp](https://github.com/10gen/mongo/blob/c176492ec7c149a081585ad52416cdc630edab81/src/mongo/db/storage/key_string.cpp#L75) for type encoding)